protocol/chainlib/chainproxy/rpcclient/http.go

```diff
@@ -163,7 +163,7 @@ func (c *Client) sendBatchHTTP(ctx context.Context, op *requestOp, msgs []*Jsonr
 	if !ok {
 		return fmt.Errorf("sendBatchHTTP - c.writeConn.(*httpConn) - type assertion failed, type:" + fmt.Sprintf("%s", c.writeConn))
 	}
-	respBody, err := hc.doRequest(ctx, msgs)
+	respBody, err := hc.doRequest(ctx, msgs, false)
```

Comment:
here true as well as this is a default jsonrpc client not tendermint.

Reply:
done: 716bc2e03e4756f6168f912bd89afd04b1a615e7